corehq/apps/users/views/__init__.py

```diff
@@ -710,6 +710,17 @@ def get_roles_for_display(self):
             )
         return role_view_data
 
+    def get_possible_profiles(self):
+        from corehq.apps.users.views.mobile.custom_data_fields import (
+            CUSTOM_USER_DATA_FIELD_TYPE,
+        )
+        return [{
+                'id': profile.id,
+                'name': profile.name,
+                }
+            for profile in CustomDataFieldsDefinition.get(self.domain, CUSTOM_USER_DATA_FIELD_TYPE).get_profiles()
```

Comment:
oops i forgot a thing in my original comment. `CustomDataFieldsDefinition.get` could return None which would error if you try to call `get_profiles()`

Reply:
good call. I didn't catch that either. adding a check.